modules/store.js

```diff
@@ -0,0 +1,24 @@
+export default class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(id) {
+    let books = Store.getBooks();
+    id = parseInt(id, 10);
+    books = books.filter((book) => book.id !== id);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
```

Comment:
- [ ] I noticed a small bug 🐛 where if you delete a book, its element gets removed from the DOM but the book still exists in `localStorage` so when you reload the page, the book appears again. Please try to fix this in order to get approved 👍